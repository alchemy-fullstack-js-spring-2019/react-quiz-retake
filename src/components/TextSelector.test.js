import React from 'react';
import { shallow } from 'enzyme';
import TextSelector from './TextSelector';

describe('TextSelector component', () => {
  it('renders the app', () => {
    const handleChange = jest.fn();
    const wrapper = shallow(<TextSelector headerText='am i' footerText='doing this right' image='maybe.jpg' handleChange={handleChange} />);
    expect(wrapper).toMatchSnapshot();
  });
});
