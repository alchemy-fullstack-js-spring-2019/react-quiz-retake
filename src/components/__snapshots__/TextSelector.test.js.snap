// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextSelector component renders the app 1`] = `
<Fragment>
  <input
    name="headerText"
    onChange={[MockFunction]}
    type="text"
    value="am i"
  />
  <input
    name="footerText"
    onChange={[MockFunction]}
    type="text"
    value="doing this right"
  />
  <input
    name="image"
    onChange={[MockFunction]}
    type="text"
    value="maybe.jpg"
  />
</Fragment>
`;
